crypto/asn1/a_int.c

```diff
@@ -82,7 +82,7 @@ static void twos_complement(unsigned char *dst, const unsigned char *src,
     dst += len;
     src += len;
     /* two's complement value: ~value + 1 */
-    for (; len > 0; len--) {
+    while (len--) {
```

Comment:
Make that `while (len-- > 0)`, it's more in line with the expliciteness that we usually use for non-boolean values.